src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Indentation/AbstractIndentation.Indenter.cs

```diff
@@ -51,8 +52,8 @@ internal abstract partial class AbstractIndentation<TSyntaxRoot>
             _syntaxFacts = service.SyntaxFacts;
             Options = options;
             Tree = tree;
-            Text = tree.GetText(cancellationToken);
```

Comment:
>Text = tree.GetText(cancellationToken); [](http://example.com/codeflow?start=12&length=39)

Made this change, as without it, the tree would end up trying to create an underlying sourcetext.